pkg/apis/compliance/v1alpha1/compliancescan_types.go

```diff
@@ -58,6 +58,8 @@ const (
 	PhaseRunning ComplianceScanStatusPhase = "RUNNING"
 	// PhaseAggregating represents the scan aggregating the results
 	PhaseAggregating ComplianceScanStatusPhase = "AGGREGATING"
+	// PhaseSuspended represents the scan schedule is on hold and will not run
+	PhaseSuspended ComplianceScanStatusPhase = "SUSPENDED"
```

Comment:
Revert this change since it makes more sense to associate the status to the binding instead of the suite. It makes it easier to short-circuit the suite creation if we keep associate the status to the binding.